ampd/src/handlers/multisig.rs

```diff
@@ -54,6 +53,7 @@ where
 pub struct Handler<S> {
     verifier: TMAddress,
     multisig: TMAddress,
+    chain: ChainName,
```

Comment:
I was actually thinking the handler would take a list of chains, instead of one handler per chain. This is interesting though, because this effectively changes the execution model, because now signing could run in parallel for several chains. Though this will dramatically increase the number of tasks. I actually prefer this model, since each chain has a dedicated voting handler, and so it makes sense to do the same for signing. 

However, I'd prefer to do this as a separate PR, so that way we don't force verifiers to update their configs as part of this release. There's already enough moving parts.

Reply:
Created [a separate PR](https://github.com/axelarnetwork/axelar-amplifier/pull/772) and [reverted the commit](https://github.com/axelarnetwork/axelar-amplifier/pull/764/commits/849e724931bf8c6a4676fa83b8c1729c73b7e02d) from here (`xrpl-ampd`).